tools/modelOrderReduction.py

```diff
@@ -31,24 +31,27 @@
 path = os.path.dirname(os.path.abspath(__file__))
 sys.path.append(path+'/../python') # TO CHANGE
 
+useGui = True
 
 # MOR IMPORT
-from mor.gui import utility
+if useGui:
+    from mor.gui import utility
+
 from mor.reduction import ReduceModel
 from mor.reduction.container import ObjToAnimate
 
 #######################################################################
 ####################       PARAMETERS       ###########################
 
 # Select Output Dir and original scene name & path
-from PyQt5 import QtWidgets
-app = QtWidgets.QApplication(sys.argv)
-
-originalScene = utility.openFileName('Select the SOFA scene you want to reduce')
-outputDir = utility.openDirName('Select the directory that will contain all the results')
-
-# originalScene = "/home/felix_v/SOFA/plugins/ModelOrderReduction/tools/test/sofa_test_scene/diamondRobot.py"
-# outputDir = "/home/felix_v/SOFA/plugins/ModelOrderReduction/tools/test/test_diamond"
+if useGui:
+    from PyQt5 import QtWidgets
+    app = QtWidgets.QApplication(sys.argv)
+    originalScene = utility.openFileName('Select the SOFA scene you want to reduce')
+    outputDir = utility.openDirName('Select the directory that will contain all the results')
+else:
+    originalScene = "/home/felix_v/SOFA/plugins/ModelOrderReduction/tools/test/sofa_test_scene/diamondRobot.py"
```

Comment:
Can you please remove my hard-coded path and change it to a generic indication ?
Sorry about that I shouldn't have committed it in the first place.
Like so for example :
```
originalScene = None # replace with path to your scene
outputDir = None # replace with path to folder where results will be
```